pages/macroalgae.js

```diff
@@ -76,7 +80,196 @@ const Index = () => {
       <Container>
         <Guide color='teal' />
       </Container>
-      <Box sx={{ position: 'absolute', top: 0, bottom: 0, width: '100%' }}>
+      <Box sx={{ position: 'absolute', top: 0, width: '100%', zIndex: 5000 }}>
+        <Container>
+          <Header />
+        </Container>
+      </Box>
+      <Container>
+        <Row>
+          <Column width={3} start={1}>
+            <Box
+              sx={{
+                position: 'absolute',
+                left: [
+                  '0px',
+                  '0px',
+                  '0px',
+                  'max(calc((100vw - 1920px)/2), 0px)',
+                ],
+                right: [
+                  'calc(3 * 100vw / 6 - 12px)',
+                  'calc(5 * 100vw / 8 - 18px)',
+                  'calc(9 * 100vw / 12 - 24px)',
+                  'calc(8 * 100vw / 12 - 24px)',
+                ],
+                zIndex: 1000,
+                transition: 'transform 0.2s',
+                transform: expanded ? 'translateX(0)' : 'translateX(-100%)',
+              }}
+            >
+              <Box
+                onClick={() => setExpanded((prev) => !prev)}
```

Comment:
Is it possible to use `as='button'` here? Would be nice if we can easily adopt the accessibility features of a button (keyboard interactions, focus state, etc.)

Reply:
ahhh good call, adding that now!